afdko/Tools/SharedData/FDKScripts/BezTools.py

```diff
@@ -23,8 +22,8 @@
 import re
 import time
 import os
-from . import FDKUtils
-from . import ConvertFontToCID
+import FDKUtils
```

Comment:
this `import FDKUtils` will fail on python 3 if it is meant to be an implicit relative import: ie. the FDKUtils.py is a module that happens to sit in the same package directory as the current module which is importing it.
It only works if FDKUtils is an absolute import (because it doesn't have a ".").
I don't know what the packages layout in afdko is meant to be.
is the current module intented to be imported as `afdko.Tools.SharedData.FDKScripts.BezTools`?

Reply:
if I run `python3 autohint` (that import bezTools and FDKUtils) it works fine

fdk is a script collection, not a package, how ever sometimes they need each other. Its not clear :)

Reply:
exactly, it works if you run `python3 autohint.py` from the same directory (the current working directory is always in the PYTHONPATH), in which case absolute or relative imports are the same thing.

Reply:
I did both `python3 full/path/to/autohint.py` as what FDK is actually doing cd'ing to the tools folder and execute from there, see https://github.com/adobe-type-tools/afdko/blob/master/afdko/Tools/osx/autohint

Reply:
I haven't looked into it, but I think the `setFDKPaths` shell script must be doing something to ensure that the FDKScripts directory is in the PYTHONPATH. In which case `import FDKUtils` does work.